Tests/UnitTests/CardTests/CardTests.swift

```diff
@@ -1,12 +0,0 @@
-import XCTest
-@testable import Card
-
-final class CardTests: XCTestCase {
```

Comment:
Replaced with `Card_Tests`, and updated appropriately. This wasn't compiling on master - I think it was a floating file / not actually running in any test suite.